src/Pages/Application/ApplicationPage.tsx

```diff
@@ -0,0 +1,24 @@
+import React,{Component} from 'react';
+import {Input, Switch, Box, Flex, Select, Textarea} from '@chakra-ui/react';
+import { useForm, UseFormRegisterReturn } from 'react-hook-form'
+
+function ApplicationPage() {
+
+    return (
+        <Flex direction='column' alignItems='center' justifyContent='center'>
+            <Input placeholder='email@berkeley.edu' borderColor="darkgrey" width="70%" height="60px" mt={5} />
+            <Textarea placeholder='Here is a larger box.' />
+            <Switch colorScheme='blue' />
+            <Select variant='outline' placeholder='Year' >
+                <option value='option1'>Freshman</option>
```

Comment:
Instead of having multiple options, use an array then map over it.

i.e.

```
const yearOptions = [
{ 
  label: 'Freshmen',
  value: 'freshmen',
], 
```
etc